### PR TITLE
[IT] Added `Gameplay/Hit_object`

### PR DESCRIPTION
**Pull request by massimoty**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Hit_object/it.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - hit objects
+---
+
+<!-- TODO: should also have articles for each hit object -->
+
+<!-- TODO: other games modes? =( -->
+
+# Note
+
+Le *note* sono elementi con cui il player puo interagire durante il gioco di una [beatmap](/wiki/Beatmap). Ogni [game mode](/wiki/Game_mode) ha il suo set unico di note che cambia la esperienza di gioco. Per una panoramica sulle note di ciascuna modalità di gioco, guarda la seguente tabella:
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:12]:
```suggestion
Le *note* sono elementi con cui il giocatore può interagire durante una [beatmap](/wiki/Beatmap). Ogni [modalità di gioco](/wiki/Game_mode) ha il suo set unico di note che cambia l'esperienza di gioco. Per una panoramica sulle note di ciascuna modalità di gioco, guarda la seguente tabella:
```

File: wiki/Gameplay/Hit_object/it.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+tags:
+  - hit objects
+---
+
+<!-- TODO: should also have articles for each hit object -->
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:6]:
```suggestion
```

File: wiki/Gameplay/Hit_object/it.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+<!-- TODO: other games modes? =( -->
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:8]:
```suggestion
```

File: wiki/Gameplay/Hit_object/it.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+tags:
+  - hit objects
+---
+
+<!-- TODO: should also have articles for each hit object -->
+
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:7]:
```suggestion
```

File: wiki/Gameplay/Hit_object/it.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+tags:
+  - hit objects
+---
+
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:5]:
```suggestion
```

File: wiki/Gameplay/Hit_object/it.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+| [osu!](/wiki/Game_mode/osu!) | [osu!taiko](/wiki/Game_mode/osu!taiko) | [osu!catch](/wiki/Game_mode/osu!catch) | [osu!mania](/wiki/Game_mode/osu!mania) |
+| :-: | :-: | :-: | :-: |
+| [cerchi](/wiki/Gameplay/Hit_object/Hit_circle) | cerchi | [frutti](/wiki/Gameplay/Hit_object/Fruit) | note |
+| [sliders](/wiki/Gameplay/Hit_object/Slider) | rullo di tamburi | [fiumi di succo](/wiki/Gameplay/Hit_object/Juice_stream) | note lunghe |
+| [spinner](/wiki/Gameplay/Hit_object/Spinner) | dendens | [banane](/wiki/Gameplay/Hit_object/Banana) | x |
+| x | x | [hyperfruits](/wiki/Gameplay/Hit_object/Hyperfruit) | x |
+
+Dal punto di vista della programmazione, le note lunghe in osu!mania non sono uguali agli slider su osu!, anche se in questa tabella sono stati raggruppati insieme per semplificare. Altri elementi con cui non si interagisce durante il gioco, come la barra della salute o le stelle kiai, sono considerati elementi che migliorano il gioco o parte dell'interfaccia utente.
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:21]:
```suggestion
Dal punto di vista della programmazione, le note lunghe in osu!mania non sono uguali agli slider su osu!, anche se in questa tabella sono stati raggruppate insieme per semplificare. Altri elementi con cui non si interagisce durante il gioco, come la barra della salute o le stelle kiai, sono considerati elementi che migliorano il gioco o parte dell'interfaccia utente.
```

File: wiki/Gameplay/Hit_object/it.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+In osu!, la prossima nota o slider viene evidenziata con una luminosità leggermente maggiore rispetto alle altre note sul campo di gioco. Tuttavia, l'evidenziazione non sarà più visibile quando la [velocità di avvicinamento](/wiki/Beatmap/Approach_rate) è troppo alta.
+
+<!-- TODO: Add links to other articles when they're available -->
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:25]:
```suggestion
```

File: wiki/Gameplay/Hit_object/it.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+| :-: | :-: | :-: | :-: |
+| [cerchi](/wiki/Gameplay/Hit_object/Hit_circle) | cerchi | [frutti](/wiki/Gameplay/Hit_object/Fruit) | note |
+| [sliders](/wiki/Gameplay/Hit_object/Slider) | rullo di tamburi | [fiumi di succo](/wiki/Gameplay/Hit_object/Juice_stream) | note lunghe |
+| [spinner](/wiki/Gameplay/Hit_object/Spinner) | dendens | [banane](/wiki/Gameplay/Hit_object/Banana) | x |
+| x | x | [hyperfruits](/wiki/Gameplay/Hit_object/Hyperfruit) | x |
+
+Dal punto di vista della programmazione, le note lunghe in osu!mania non sono uguali agli slider su osu!, anche se in questa tabella sono stati raggruppati insieme per semplificare. Altri elementi con cui non si interagisce durante il gioco, come la barra della salute o le stelle kiai, sono considerati elementi che migliorano il gioco o parte dell'interfaccia utente.
+
+In osu!, la prossima nota o slider viene evidenziata con una luminosità leggermente maggiore rispetto alle altre note sul campo di gioco. Tuttavia, l'evidenziazione non sarà più visibile quando la [velocità di avvicinamento](/wiki/Beatmap/Approach_rate) è troppo alta.
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:23]:
```suggestion
In osu!, la prossima nota o slider viene evidenziata con una luminosità leggermente più altra rispetto alle altre sul campo di gioco. Tuttavia, l'evidenziazione non sarà più visibile quando la [velocità di avvicinamento](/wiki/Beatmap/Approach_rate) è troppo alta.
```

**Review by Sclafus — CHANGES_REQUESTED** [review]

**Review by Sclafus — APPROVED** [review]

File: wiki/Gameplay/Hit_object/it.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+---
+tags:
+  - hit objects
+---
+# Note
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:5]:
```suggestion

# Note
```

File: wiki/Gameplay/Hit_object/it.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+| :-: | :-: | :-: | :-: |
+| [cerchi](/wiki/Gameplay/Hit_object/Hit_circle) | cerchi | [frutti](/wiki/Gameplay/Hit_object/Fruit) | note |
+| [sliders](/wiki/Gameplay/Hit_object/Slider) | rullo di tamburi | [fiumi di succo](/wiki/Gameplay/Hit_object/Juice_stream) | note lunghe |
+| [spinner](/wiki/Gameplay/Hit_object/Spinner) | dendens | [banane](/wiki/Gameplay/Hit_object/Banana) | x |
+| x | x | [hyperfruits](/wiki/Gameplay/Hit_object/Hyperfruit) | x |
+
+Dal punto di vista della programmazione, le note lunghe in osu!mania non sono uguali agli slider su osu!, anche se in questa tabella sono stati raggruppati insieme per semplificare. Altri elementi con cui non si interagisce durante il gioco, come la barra della salute o le stelle kiai, sono considerati elementi che migliorano il gioco o parte dell'interfaccia utente.
+
+In osu!, la prossima nota o slider viene evidenziata con una luminosità leggermente più alta rispetto alle altre sul campo di gioco. Tuttavia, l'evidenziazione non sarà più visibile quando la [velocità di avvicinamento](/wiki/Beatmap/Approach_rate) è troppo alta.
+
```
Comment (Sclafus) [inline wiki/Gameplay/Hit_object/it.md:19]:
```suggestion
```

**Review by Sclafus — CHANGES_REQUESTED** [review]
CI fix

**Review by Sclafus — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**